### PR TITLE
eventcomment3

### PR DESCRIPTION
**Pull request by JesseGreat**
Implemented an API endpoints that allows users to retrieve comments associated with a specific event

GET /api/events/:eventId/comments: Get comments for an event
POST /api/comments/:commentId/images: Add an image to a comment
GET /api/comments/:commentId/images: Get images for a comment

with the following Acceptance Criteria:
A new API endpoint (GET /api/events/:eventId/comments) for fetching comments for an event.
Accept the unique event identifier (eventId) as a URL parameter.
Retrieve all comments associated with the specified event from the database.
Return an appropriate response with a list of comments, including their content, creation timestamp, and user information.
Handle cases where the requested eventId does not exist and return an error response with an appropriate status code (e.g., 404 Not Found).

please reach out to me asap for any modification, my head aches abeg

### CODE REVIEW

**Comment by F0laf0lu** [comment]
Welldone boss, but there's a comment model already so you don't need to create another one

**Comment by JesseGreat** [comment]
> Welldone boss, but there's a comment model already so you don't need to create another one

Damn it I didn't take note can I edit it on GitHub here??? I think the mistake is that I didn't open all the files b4 I start my own work

**Comment by JesseGreat** [comment]
> Welldone boss, but there's a comment model already so you don't need to create another one

Do you have chance please let's resolve it together

**Comment by olatunde2** [comment]
it advisable to check the latest update before you push your code am sorry i can't be of help

File: events/views.py [hunk 1]
```diff
@@ -1,3 +1,19 @@
...
+class CommentViewSet(viewsets.ModelViewSet):
+    serializer = CommentSerialzer
+
+    def queryset(self):
+        event_id = self.kwargs['eventId']
+        try:
+            event = GroupEvents.objects.get(event_id=event_id)
+            return Comment.objects.filter(event=event)
+        except GroupEvents.DoesNotExist:
+            return Comment.objects.none()
```
Comment (denscholar) [inline events/views.py:19]: I had to close this pull request considering the fact that some other person has done the same task. You might want pull and update your code base and if you're going to make any necessary update in the code, you can do it on your local machine. 

Weldone!

**Review by denscholar — COMMENTED** [review]
check comment.

**Pull request closed without merging**